### ISSUE TITLE
Control the number of Columns per Breakpoint

### ISSUE THREAD

**Issue by blizam** [gantry/gantry5#1805]
Feature Request:

The ability to define the number of columns per view in the Gantry Grid System

It'd be killer if you could specify the number of columns per screen size with "0" being infinite. Most useful would be the ability to force 2 columns or more (instead of 1 column/100% width/stacked) on mobiles.

Something like the attached

![2017-01-16_9-13-39](https://cloud.githubusercontent.com/assets/8940155/21990711/0db39b30-dbcd-11e6-83ad-4218d5576a06.png)

**Comment by blizam** [comment]
Further or in-addition to this it would be sweet if you could specify column numbers per Section even... as a Section Setting on your Layout Manager

**Comment by mahagr** [comment]
I don't think that number of columns should be forced on this level; it should really be decided inside of layout manager per case basis. I think you could already do this by assigning an additional class and creating CSS rule for it. Something like `.mobile-50` could work..?
